__tests__/app.test.js

```diff
@@ -13,13 +13,13 @@ describe('rammevaerk:app', () => {
 
     it('generates expected files', () => {
         assert.file([
-            '.babelrc',
             '.browserslistrc',
             '.editorconfig',
             '.eslintrc',
             '.gitattributes',
             '.gitignore',
             '.stylelintrc',
+            'babel.config.js',
```

Comment:
What's the benefit of having a config in a JS file? I thought it was kind of convenient and aligned to have this stuff in `rc` files (e.g. ESLint, stylelint, Browserslist). Alternatively config could be moved to `package.json`. Thanks 😄

Reply:
I wanted to keep the rc file for exactly the reason you mentioned. However, as I understand the config.js file is the way to introduce a [project wide config](https://babeljs.io/docs/en/config-files#project-wide-configuration). I've had the need of that on few projects and it seems like that's the that to go. Do correct me if I am wrong. :D

Reply:
Oh, now I get it... I read docs a couple of times and I couldn't think of a use case. Now I can see why it's needed: if we `import` Bootstrap from `node_modules` and process it with Babel, then our `babel.config.js` will be used. Am I right or you had another use case?

Reply:
👍 Yep that's surely the use case :)

Reply:
Perhaps this can be reverted as well now that we aren't importing bootstrap src or should we leave it as is. I see no harm but I would really like it was dot file 💭

Reply:
> I would really like it was dot file

Me too 👍 So let's just revert it back to `.babelrc` and we're good to go 😄

Reply:
@racse1 Reverted 👍 😀